applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.hpp

```diff
@@ -98,23 +131,62 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwLysmerAbsorbingCondition : public
     // Member Variables
     
 //----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-    void CalculateDampingMatrix(MatrixType& rDampingMatrix, const ProcessInfo& CurrentProcessInfo) override;
 
-    void CalculateLocalSystem(MatrixType& rLhsMatrix, VectorType& rRightHandSideVector, const ProcessInfo& CurrentProcessInfo) override;
-    
-    void CalculateAndAddLHS(MatrixType& rLeftHandSideMatrix, NormalLysmerAbsorbingVariables& rVariables);
-
-    void CalculateRotationMatrix(BoundedMatrix<double, TDim, TDim>& rRotationMatrix, const Element::GeometryType& Geom);
-
-    void GetNeighbourElementVariables(NormalLysmerAbsorbingVariables& rVariables, const ProcessInfo& CurrentProcessInfo);
-
-    void GetVariables(NormalLysmerAbsorbingVariables& rVariables, const ProcessInfo& CurrentProcessInfo);
 
-    void CalculateNodalDampingMatrix(NormalLysmerAbsorbingVariables& rVariables, const ProcessInfo& CurrentProcessInfo, const Element::GeometryType& Geom);
-
-    void CalculateNodalStiffnessMatrix(NormalLysmerAbsorbingVariables& rVariables, const ProcessInfo& CurrentProcessInfo, const Element::GeometryType& Geom);
-
-    Matrix CalculateExtrapolationMatrixNeighbour(const Element& NeighbourElement);
+    /**
+    * @brief Adds the condition matrix to the global left hand side
+    * @param rLeftHandSideMatrix Global Left hand side
+    * @param rUMatrix LHS displacement matrix of the current condition
+    */
+    void AddLHS(MatrixType& rLeftHandSideMatrix, const BoundedMatrix<double, N_DOF, N_DOF>& rUMatrix);
```

Comment:
Just a minor suggestion. It seems that you are using `BoundedMatrix<double, N_DOF, N_DOF>` in many places. Perhaps, you can define it in the header file as `typedef BoundedMatrix<double, N_DOF, N_DOF> ElementMatrixType;`. Another type is `typedef BoundedMatrix<double, TDim, TDim> RotationMatrixType;`

Reply:
thanks, I've changed it